### PR TITLE
#160452438 User should be able to create an article

### PR DESCRIPTION
**Pull request by Alameen688**
#### What does this PR do?
Implements create user form design and allow a user to create an article
#### Description of Task to be completed?
- Include current design for the create article page
- Add publish settings to include tags
- Allow a user to create an article
#### How should this be manually tested?
- Checkout to this branch `git checkout ft-160452438-create-article`
- Navigate to the axio.js file in the services directory, set the value of the token constant to a valid user token
- RUN `npm start`
- Navigate to [http://localhost:3000/article/add](http://localhost:3000/article/add)
#### Screenshots (if appropriate)
screen shot 2018-11-03 at 10 23 02 am

#### Questions:

### CODE REVIEW

File: src/components/article/createArticle/createArticle.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import validate from 'validate.js';
+import { createArticleRequest } from '../../../redux/actions/articles/articles';
+import { createArticleConstraint } from '../../../utils/validator/constraint/article';
+import './createArticle.scss';
+
+export class CreateArticle extends Component {
+  state = {
```
Comment (waleCloud) [inline src/components/article/createArticle/createArticle.js:24]: Kindly take state management to redux.

**Review by waleCloud — COMMENTED** [review]
Increase test coverage, endeavor to cover all necessary files created/modified. before raising a final PR.
well done so far! 👍

File: src/components/article/createArticle/createArticle.js (hunk 1 above)
Comment (Alameen688) [reply src/components/article/createArticle/createArticle.js:24]: Alright, I will do that. Thank you.

**Comment by Alameen688** [comment]
This PR no longer needs to be raised separately against ft-160452438-create-update-article from ft-160452438-create-article. The branch ft-160452438-create-update-article now contains all required file contents to be directly raised against staging.

**Pull request closed without merging**